noggin/controller/user.py

```diff
@@ -80,9 +86,8 @@ def user_settings_profile(ipa, username):
                 'fasgitlabusername': form.gitlab.data.lstrip('@'),
                 'fasrhbzemail': form.rhbz_mail.data,
             },
+            "user_settings_profile",
         )
-        if result:
-            return result
```

Comment:
If you're not returning the redirect that `_user_mod` returned, then is it still redirecting?